figures/S9.supplemental_figure9/scripts/S9.r

```diff
@@ -203,6 +205,7 @@ fig5_probabilities <- (
     + ridge_plot_apoptosis
     + ridge_plot_pyroptosis
     + plot_layout(design = layout)
+    + plot_annotation(tag_levels = "A") &  theme(plot.tag = element_text(size = 20))
```

Comment:
looks like you need to remove this line (duplicated in line 209)

Reply:
Wrong branch...